src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,23 @@
+import React, { render, screen} from '@testing-library/react';
+import SearchResults from "../components/SearchResults";
+
+describe("Search", () => {
+    const validProps = {
+        searchResults: ["moon", "venus"]
+    };
+
+    it("renders search result component correctly", () => {
+        const { asFragment } = render(
+            <SearchResults searchResults={validProps.searchResults} />);
+            expect(asFragment()).toMatchSnapshot();
+        });
+
+        it("renders placeholder if no results returned", () => {
+            const validProps = [];
+            render(<SearchResults results={validProps} />);
```

Comment:
This test is broken. The prop being passed in is `result` and not `searchResult` as it should be so is coming through as undefined